pyramid/session.py

```diff
@@ -215,7 +215,7 @@ def check_csrf_token(request,
     supplied by ``request.session.get_csrf_token()``, and ``raises`` is
     ``True``, this function will raise an
     :exc:`pyramid.exceptions.BadCSRFToken` exception.
-    If the check does succeed and ``raises`` is ``False``, this
+    If the check does not succeed and ``raises`` is ``False``, this
```

Comment:
I think the term "does succeed" could be more clear. I had to read this statement several times and look at the code closely. In this context I think "does succeed" means `strings_differ(bytes_(expected_token), bytes_(supplied_token))` is `True`.

Anyway, I think that a better phrase would be the following:

```rst
If the values differ and ``raises`` is ``False``, this
function will return ``False``.
```

I'd like one of the other core developers to double-check this.